tests/helpers/task_tester.py

```diff
@@ -102,8 +102,10 @@ def _test_jit_trace(self, tmpdir):
     path = os.path.join(tmpdir, "test.pt")
 
     model = self.instantiated_task
+    trainer = self.instantiated_trainer
     model.eval()
 
+    model.trainer = trainer
```

Comment:
See https://github.com/Lightning-AI/lightning/issues/14036#issuecomment-1206643603 for a workaround

Reply:
Just curious, anything wrong with the current workaround?